roles/openshift_node/tasks/upgrade/config_changes.yml

```diff
@@ -74,4 +75,6 @@
 #       require a service to be part of the call.
 - name: Reload systemd units
   command: systemctl daemon-reload
-  when: l_node_unit is changed
+  when:
```

Comment:
Let's get rid of this when condition here and just always reload it.

Reply:
Done